From f2abddb8b60c009bd95992371a9532a6e60c5a91 Mon Sep 17 00:00:00 2001
From: Matt Johnston <matt@ucc.asn.au>
Date: Wed, 22 Jul 2026 19:23:31 +0800
Subject: Fix const strchr warning

Some compilers treat strchr() return value as const char*
No functional change.
---
 src/cli-runopts.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/cli-runopts.c
+++ b/src/cli-runopts.c
@@ -1138,7 +1138,7 @@ static void add_extendedopt(const char*
 
 #if DROPBEAR_USE_SSH_CONFIG
 static void apply_config_settings(const char* cli_host_arg) {
-	char* is_multi_hop_host_target = strchr(cli_host_arg, ',');
+	const char* is_multi_hop_host_target = strchr(cli_host_arg, ',');
 	if (!is_multi_hop_host_target) {
 		char* config_path = expand_homedir_path(DROPBEAR_DEFAULT_SSH_CONFIG);
 		FILE* f;
